content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -353,6 +352,21 @@ The `PodTopologySpread` plugin does not score the nodes that don't have
 the topology keys specified in the spreading constraints.
 {{< /note >}}
 
+If a default spreading is not desired in your cluster, you can disable it in the
+`PodTopologySpread` plugin args:
```

Comment:
Could you provide a bit more detail? Do you set the `defaultConstraints` to an empty list?
`If a default spreading topology is not desired in your cluster, you can disable it by setting the `defaultConstraints` to ... ?

Reply:
See update